interpreters/src/tests.rs

```diff
@@ -115,6 +115,49 @@ where
         }
     }
 
+    async fn test_insert_table_with_missing_columns(&self) {
+        let catalog_manager = Arc::new(build_catalog_manager(self.engine()).await);
+        let ctx = Context::builder(RequestId::next_id())
+            .default_catalog_and_schema(DEFAULT_CATALOG.to_string(), DEFAULT_SCHEMA.to_string())
+            .build();
+        let insert_factory =
+            Factory::new(ExecutorImpl::new(), catalog_manager.clone(), self.engine());
+        let insert_sql = "INSERT INTO test_missing_columns_table(key1, key2) VALUES('tagk', 1638428434000), ('tagk2', 1638428434000);";
+
+        let plan = sql_to_plan(&self.meta_provider, insert_sql);
+        let interpreter = insert_factory.create(ctx, plan);
+        let output = interpreter.execute().await.unwrap();
+        if let Output::AffectedRows(v) = output {
```

Comment:
Do you mean to use `ensure!(matches!(output, Output::AffectedRows(v) if v == 2, Context)` here? 
Seems `ensure!` is not suitable, because the result of this function is a unit type, not a `Result`. 

Maybe `assert!` and `matches!` is enough.

```rust
assert!(
     matches!(output, Output::AffectedRows(v) if v == 2),
     "insert should success"
);
```